File: enternumber.c
#include <stdio.h>

int main(){

  float a=0,b=0;

  printf("Enter two numbers:\n");

  //read the input from the user

  scanf("%f %f",&a, &b); //store at the address of num

  printf("The sum is: %010.2f+%d=%.4f\n",a, (int) b,a+b);

  //& is the "address of" operator
  //
}

File: EnterNumber.java
import java.util.Scanner;


public class EnterNumber{

    public static void main(String args[]){

	int num;
	Scanner sc = new Scanner(System.in); // standard input (stdin)

	System.out.println("Enter a number:"); //standard output (stdout)
	num = sc.nextInt(); //read in that input, as an integer, and return
	
	System.out.println("You entered: "+num); //standard output
	/*                  \  string    / + integer       
			    automatically converted to the output you want
	 */
	
    }

}
